shared/common-adapters/choice-list.desktop.tsx

```diff
@@ -11,17 +12,17 @@ const ChoiceList = ({options}: Props) => {
         // TODO is this okay?
         const iconType: IconType = op.icon as IconType
         return (
-          <Box style={styleEntry} key={idx} className="cl-entry" onClick={() => op.onClick()}>
-            <Box style={styleIconContainer} className="cl-icon-container">
+          <Box style={Styles.collapseStyles([Styles.globalStyles.flexBoxRow, Styles.desktopStyles.clickable, styles.entry])} key={idx} className="cl-entry" onClick={() => op.onClick()}>
```

Comment:
Can you make these into `Box2`s (which are flexbox automatically, among other things) while you're here?

Reply:
let's not do this. we're going to touch hundreds of files and we can't stop and refactor things, especially the common-adapters where it could break a ton of components